db/table_cache.cc

```diff
@@ -444,6 +528,31 @@ Status TableCache::MultiGet(const ReadOptions& options,
     }
   }
 
+#ifndef ROCKSDB_LITE
+  if (lookup_row_cache) {
+    size_t row_idx = 0;
```

Comment:
I think this could also be factored out into a helper method and reused in Get/MultiGet.